tests/Appsec/Mock.php

```diff
@@ -87,74 +88,132 @@ public function getEvents(array $names = [], array $addresses = [])
 }
 
 if (!function_exists('datadog\appsec\appsecMockEnabled')) {
-    function appsecMockEnabled() {
+    function appsecMockEnabled()
+    {
         return getenv('APPSEC_MOCK_ENABLED') === "true";
     }
 }
 
+if (!function_exists('datadog\appsec\track_user_login_success_event_automated')) {
```

Comment:
you should write integrations tests in appsec to ensure this is working with the actual appsec module